examples/md-flexible/src/domainDecomposition/RegularGridDecomposition.cpp

```diff
@@ -39,6 +39,12 @@ RegularGridDecomposition::RegularGridDecomposition(const MDFlexConfig &configura
 
   _decomposition = DomainTools::generateDecomposition(_subdomainCount, configuration.subdivideDimension.value);
 
+  double maxSigma{0};
+  for (auto sigmaMapElement : configuration.sigmaMap.value) {
+    maxSigma = std::max(maxSigma, sigmaMapElement.second);
+  }
```

Comment:
```suggestion
  double maxSigma{0};
  for (const auto &[_, sigma] : leMap) {
    maxSigma = std::max(maxSigma, sigma);
  }
```

Imho it would be even better to wrap that loop into a lambda function so that `maxSigma` can be turned into a `const` but since the function ends soon after this I'm fine.